gulp-tasks/test.js

```diff
@@ -0,0 +1,40 @@
+const gulp = require('gulp');
+const mocha = require('gulp-spawn-mocha');
+const path = require('path');
+
+const constants = require('./utils/constants');
+const packageRunnner = require('./utils/package-runner');
+const logHelper = require('./utils/log-helper');
+const pkgPathToName = require('./utils/pkg-path-to-name');
+
+const runBundledTests = (packagePath, env) => {
+  logHelper.log(`Running Test Bundles for ` +
+    `${logHelper.highlight(pkgPathToName(packagePath))}.`);
+
+  const bundleTestDirectory = path.join(
+    packagePath, 'test', constants.BUNDLE_BUILD_DIRNAME);
+  return gulp.src(path.join(bundleTestDirectory, env, '*.js'), {read: false})
+  .pipe(mocha());
+};
+
+gulp.task('test:node', gulp.series(
+  packageRunnner(runBundledTests, 'node'),
```

Comment:
(Asked, while ignorant to gulp 4 best practices:)

Is there an advantage to a single-function series vs. doing

```js
gulp.task('test:node', () => packageRunnner(runBundledTests, 'node'));
```

Or is this just a placeholder for more sub-tasks to come?